packages/enzyme-test-suite/test/RSTTraversal-spec.jsx

```diff
@@ -97,71 +68,59 @@ describe('RSTTraversal', () => {
       expect(nodeHasProperty(node, 'title')).to.equal(false);
     });
 
-    it('should parse false as a literal', () => {
+    it('should parse false', () => {
       const node = $(<div foo={false} />);
 
-      expect(nodeHasProperty(node, 'foo', 'false')).to.equal(true);
+      expect(nodeHasProperty(node, 'foo', false)).to.equal(true);
     });
 
-    it('should parse false as a literal', () => {
-      const node = $(<div foo />);
-
-      expect(nodeHasProperty(node, 'foo', 'true')).to.equal(true);
-    });
-
-    it('should parse numbers as numeric literals', () => {
-      expect(nodeHasProperty(<div foo={2.3} />, 'foo', '2.3')).to.equal(true);
-      expect(nodeHasProperty(<div foo={2} />, 'foo', '2')).to.equal(true);
-      expect(() => nodeHasProperty(<div foo={2} />, 'foo', '2abc')).to.throw();
-      expect(() => nodeHasProperty(<div foo={2} />, 'foo', 'abc2')).to.throw();
-      expect(nodeHasProperty(<div foo={-2} />, 'foo', '-2')).to.equal(true);
-      expect(nodeHasProperty(<div foo={2e8} />, 'foo', '2e8')).to.equal(true);
-      expect(nodeHasProperty(<div foo={Infinity} />, 'foo', 'Infinity')).to.equal(true);
-      expect(nodeHasProperty(<div foo={-Infinity} />, 'foo', '-Infinity')).to.equal(true);
+    it('should parse numeric literals', () => {
+      expect(nodeHasProperty(<div foo={2.3} />, 'foo', 2.3)).to.equal(true);
+      expect(nodeHasProperty(<div foo={2} />, 'foo', 2)).to.equal(true);
+      expect(nodeHasProperty(<div foo={2} />, 'foo', '2abc')).to.equal(false);
+      expect(nodeHasProperty(<div foo={2} />, 'foo', 'abc2')).to.equal(false);
+      expect(nodeHasProperty(<div foo={-2} />, 'foo', -2)).to.equal(true);
+      expect(nodeHasProperty(<div foo={2e8} />, 'foo', 2e8)).to.equal(true);
+      expect(nodeHasProperty(<div foo={Infinity} />, 'foo', Infinity)).to.equal(true);
+      expect(nodeHasProperty(<div foo={-Infinity} />, 'foo', -Infinity)).to.equal(true);
     });
 
     it('should parse zeroes properly', () => {
-      expect(nodeHasProperty(<div foo={0} />, 'foo', '0')).to.equal(true);
-      expect(nodeHasProperty(<div foo={-0} />, 'foo', '-0')).to.equal(true);
-      expect(nodeHasProperty(<div foo={1} />, 'foo', '0')).to.equal(false);
-      expect(nodeHasProperty(<div foo={2} />, 'foo', '-0')).to.equal(false);
+      expect(nodeHasProperty(<div foo={0} />, 'foo', 0)).to.equal(true);
+      expect(nodeHasProperty(<div foo={0} />, 'foo', +0)).to.equal(true);
+      expect(nodeHasProperty(<div foo={-0} />, 'foo', -0)).to.equal(true);
```

Comment:
can we add test cases that `-0` does not match `0`, and `0` does not match `-0`?

Reply:
Done.